<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="text_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mark_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mark_kind_cb">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="collocations_tb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
